### DISCUSSION TITLE
Dask support

### DISCUSSION THREAD

**Discussion by n0tools** [lenskit/lkpy#288] [Q&A]
Hello

I saw up to 0.8.4 reccomender batch could return a dask data frame

Has this been dropped now?

**Comment by mdekstrand** [comment]
Yes, newer versions of LensKit no longer support Dask in `batch.recommend` (or anywhere else - there might be a few places left where it works, but it is not tested). The code was adding complexity to both implementation and testing, and I had yet to find it useful in my group's work (and hadn't heard from anyone actually using it).

It could be useful to build a version of `batch.recommend` that uses Dask, and the model sharing infrastructure combined with Binpickle's shared library mode could probably make that work relatively well. It would be best implemented as a separate function, though: a Dask-backed replacement for the LensKit `batch` routines.
